release-notes/cluster-observability-operator-release-notes.adoc

```diff
@@ -26,6 +26,156 @@ The following table provides information about which features are available depe
++
+With this release, the {coo-full} troubleshooting panel correctly sanitizes Kubernetes label names by replacing invalid characters with underscores when generating Loki queries. As a result, users can successfully access logs from the correlation graph for all pods regardless of their label names.
++
+link:https://issues.redhat.com/browse/COO-1278[COO-1278]
+////
+
+////
+Troubleshooting panel navigates to Kubernetes Events page::
+Before this update, clicking on Kubernetes Events in the troubleshooting panel visualization node failed to navigate to the Events page in the {product-title} console. As a consequence, users could not view detailed Kubernetes Event information through the troubleshooting panel workflow and had to manually navigate to the Events page.
++
+With this release, the troubleshooting panel correctly handles navigation to the Kubernetes Events page when users click on event nodes in the visualization. As a result, users can successfully navigate from the troubleshooting panel to view detailed Kubernetes Event information in the console.
++
+link:https://issues.redhat.com/browse/COO-1178[COO-1178]
+////
+
+////
+Troubleshooting panel navigates to VirtualMachine resources correctly::
+Before this update, the troubleshooting panel generated incorrect URLs when navigating to `VirtualMachineInstance` resources from the visualization graph. The panel used an invalid resource path format that did not exist in the {product-title} console. As a consequence, users received 404 errors when clicking on `VirtualMachineInstance` nodes in the troubleshooting panel and could not view virtual machine details.
++
+With this release, the troubleshooting panel generates correct URLs using the proper API group format (`kubevirt.io~v1~VirtualMachine`) when navigating to virtual machine resources. As a result, users can successfully navigate from the troubleshooting panel to view `VirtualMachine` resource details in the console.
++
+link:https://issues.redhat.com/browse/COO-1219[COO-1219]
+////
+
+Korrel8r runs on FIPS-enabled clusters::
+Before this update, the korrel8r binary was not compiled with Federal Information Processing Standards (FIPS) compliant mode enabled. As a consequence, korrel8r pods failed to start on FIPS-enabled clusters and logged the error "FIPS mode is enabled, but this binary is not compiled with FIPS compliant mode enabled."
++
+With this release, the korrel8r container image uses a FIPS-compliant base image and the binary is compiled with FIPS mode enabled. As a result, korrel8r runs successfully on FIPS-enabled {product-title} clusters.
++
+link:https://issues.redhat.com/browse/COO-555[COO-555]
```

Comment:
While correct, I'd argue we leave this out. COO as a whole is not FIPS compliant, so on component being FIPS compliant doesn't change the fact that running COO will taint a FIPS cluster currently.